### DISCUSSION TITLE
能否增加查看题目和答案（可限制100%后）或者重置进度的功能

### DISCUSSION THREAD

**Discussion by shawti** [AmazingAng/WTF-Solidity#350] [Q&A] [answered]
如题
当测试通过以后，是否可以增加一个查看题目和答案或者重置进度的功能，用于过一段时间之后的复习
有些题目可能属于易错题，在反复纠错后尝试通过。过一段时间再做可能还会错，但进度已经100%，无法确定重做的题是否正确。

**Comment by liwuhou** [comment]
感觉确实有必要，这几天用下来，经常就错一道题，然后我还不知道具体是哪道题错了，为了达到 100%，纠错难度不亚于排查 bug

**Comment by XdpCs** [comment, marked as answer]
https://github.com/WTFAcademy/frontend 可以提交在此仓库下，关于前端的一些意见。
